.gitignore

```diff
@@ -1,12 +1,25 @@
 # MacOS Build Artifacts
 *.DS_Store
-# Don't commit .env files with secret info, EVER
+
+# Secrets and keys ****DO NOT COMMIT SECRETS***
```

Comment:
I think it's fine to have this stuff in the root-level `.gitignore`, but we should _also_ have a `.gitignore` with the env/secret/log stuff in the `atlas-sdk-go` directory so it gets copied into the copied app and out to the artifact repo.

Reply:
good catch. it got caught in my overly loose ignore pattern for hidden files. I'll update it and make sure BH copies it.